plugins/CoreHome/templates/_siteSelectHeader.twig

```diff
@@ -1,3 +1,3 @@
 <div class="top_bar_sites_selector piwikTopControl">
-    <div piwik-siteselector show-selected-site="true" class="sites_autocomplete"></div>
+    <div piwik-siteselector show-selected-site="true" show-all-sites-item="{{ (isMultiSitesEnabled|default) ? 'true' : 'false' }}" class="sites_autocomplete"></div>
```

Comment:
...and the [default filter](https://twig.symfony.com/doc/2.x/filters/default.html) won't be necessary:
```twig
<div
    class="sites_autocomplete"
    piwik-siteselector show-selected-site="true"
    show-all-sites-item="{{ isMultiSitesEnabled ? 'true' : 'false' }}">
</div>
```

Reply:
I had the `default` in there as I couldn't guarantee the `basic variables` method in the controller is always called so just wanted to be sure to not break anything. I could remove it as it's likely not needed. It just means if someone did like `$view = new View('...'); $view->render()` without calling the basic variables in the controller then things could break when embedding that template. From what I see it shouldn't be needed though. Will check again.

Reply:
From what I can see it can be removed indeed cheers 👍  will wait for the tests to confirm it

Reply:
Great! :)